buildinfo_test.go

```diff
@@ -737,6 +804,15 @@ func testBuildAddGit(t *testing.T, useEnvBuildNameAndNumber bool) {
 	cleanArtifactoryTest()
 }
 
+func calculateBuildNumberFrequency(runs *buildinfo.BuildRuns) map[string]int {
```

Comment:
Perhaps we can use the utility from `jfrog-cli-artifactory` to avoid duplication